CMakeLists.txt

```diff
@@ -1,34 +1,27 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(urg_c)
 
+# Support C++14
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}" -std=c++14)
```

Comment:
I'll suggest that you use the more standard template that we've been using in ROS 2 for setting this, like: https://github.com/ros2/demos/blob/857c436e76bc6b40d32f096f6989bf51bb73f359/image_tools/CMakeLists.txt#L5

Reply:
done in https://github.com/ros-drivers/urg_c/pull/11/commits/8fd5611deb0cfca0a848d63c9cad70a7844401e7